spec/unit/filter_spec.rb

```diff
@@ -2,12 +2,46 @@
 
 describe Filter do
   let(:definition1) { "/unmanaged_files/files/name=/home/alfred" }
-  let(:definition2) { "/unmanaged_files/files/name=/home/alfred,/var/cache" }
+  let(:definition2) { "\"/unmanaged_files/files/name=/home/alfred,/var/cache\"" }
```

Comment:
We defined the syntax in the design document a little bit different. In this form it would match an element in the JSON where the value is an array of `["/home/alfred", "/var/cache"]` according to the spec. But the syntax you use here is a shortcut to having to filter deinitions `/unmanaged_files/files/name=/home/alfred` and `/unmanaged_files/files/name=/var/cache`, right?

At the current point in time I wouldn't introduce these kind of shortcuts, but first get the basic filtering implemented how we planned, including command line interface and persistent storage. Then we are in a better position to judge what is needed and practical to improve the syntax.